sdk/python/kfp/dsl/_container_op.py

```diff
@@ -687,7 +687,7 @@ class BaseOp(object):
     # in the compilation process to generate the DAGs and task io parameters.
     attrs_with_pipelineparams = [
         'node_selector', 'volumes', 'pod_annotations', 'pod_labels',
-        'num_retries', 'init_containers', 'sidecars', 'tolerations'
+        'init_containers', 'sidecars', 'tolerations'
```

Comment:
Just curious: can `_node_retry_strategy` contains pipeline params?

Reply:
I'm not sure it should.
I'm aware that the change here is effectively dropping support for dynamic number of retries.
If someone complains, I'll take notice and maybe re-enable it.